kong/tools/queue_schema.lua

```diff
@@ -8,46 +8,46 @@ return Schema.define {
       type = "integer",
       default = 1,
       between = { 1, 1000000 },
-      -- description = "maximum number of entries that can be processed at a time"
+      description = "maximum number of entries that can be processed at a time"
```

Comment:
I think these descriptions should start in Uppercase and end in a dot.

```suggestion
      description = "Maximum number of entries that can be processed at a time."
```